providers/class-two-factor-backup-codes.php

```diff
@@ -347,7 +347,11 @@ public function authentication_page( $user ) {
 	 * @return boolean
 	 */
 	public function validate_authentication( $user ) {
-		$backup_code = isset( $_POST['two-factor-backup-code'] ) ? sanitize_text_field( wp_unslash( $_POST['two-factor-backup-code'] ) ) : '';
+		$backup_code = $this->sanitize_code_from_request( 'two-factor-backup-code' );
```

Comment:
Should this and email pass the `$length` param too?

Reply:
Ideally yes, however, there isn't a defined class constant or property for those right now so I simply skipped over them in the initial code.

I was considering adding a `get_length()` method to the classes, which defaulted to 8 with a filter, to allow others to adjust the length of codes, but seemed out of the scope of this. I also have questions around how to handle things like Backup Codes if Two Factor (or another plugin) were to extend the length of those later on - what would happen to existing backup codes?

Reply:
Yeah, those are good points. I think it's fine to leave as-is 👍🏻